### ISSUE TITLE
Making an Interest to Express

### ISSUE THREAD

**Issue by justincpresley** [named-data/python-ndn#26]
In one application that I am forming using this python NDN library, it is necessary to measure the size of the interest before expressing it. To accomplish this, I plan to make an 'Interest' and then express it after created manually.

I noticed though that there is no way to express a Interest that is created manually, you have to let the library create it through "express_interest".

```
name = Name.from_str("/data/section/1")
interest = make_interest(name, InterestParam(can_be_prefix=True, must_be_fresh=True, lifetime=6000))
                        # also where is need_raw_packet? app_params?
_, _, _, pkt = await app.express_interest(interest)
```

is there a method to decode a manually created Interest so that "express_interest" can use it?

**Comment by zjkmxy** [comment]
Let me write a function for you.

**Comment by zjkmxy** [comment]
Should be fixed in 00fdfd9. Reopen this issue if it does not work.
Please contact me when you need a release.
